Composer/packages/client/src/pages/design/index.tsx

```diff
@@ -312,8 +325,14 @@ function DesignPage(props) {
   }, [dialogs, breadcrumb]);
 
   async function onSubmit(data: { name: string; description: string }) {
-    const content = { ...getNewDesigner(data.name, data.description), generator: `${data.name}.lg` };
-    const seededContent = seedNewDialog('Microsoft.AdaptiveDialog', content.$designer, content);
+    const seededContent = seedNewDialog(
+      'Microsoft.AdaptiveDialog',
+      { name: data.name, description: data.description },
+      {
+        generator: `${data.name}.lg`,
+      }
+    );
+    set(seededContent, 'triggers[0].actions', state.actionsSeed || []);
```

Comment:
Why not just add this in the `seedNewDialog` call?

Reply:
Chris had a solution that we can use function inside `initialDialogShape`
```ts
[SDKTypes.AdaptiveDialog]: (initialActions) => ({
    triggers: [
      { actions: initialActions }
    ]
})
```
I just noticed that it's lost after my commit, I will recover that

Reply:
resolved